Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -45,16 +45,21 @@ public class DafnyProgramVerifier : IProgramVerifier {
           // https://github.com/boogie-org/boogie/blob/b03dd2e4d5170757006eef94cbb07739ba50dddb/Source/VCGeneration/Couterexample.cs#L217
           DafnyOptions.O.ModelViewFile = "-";
           DafnyOptions.O.VcsCores = GetConfiguredCoreCount(options.Value);
+          DafnyOptions.O.VerifySnapshots = (int)options.Value.VerifySnapshots;
```

Comment:
Is this PR still a draft? If so, mark is as such. It seems that you are defining an option but never using it anywhere except in loggin.

Reply:
It's not a draft, it's ready to go.

Reply:
What does it enable that was not possible before?

Reply:
The option is set through the command line, and the command line flag is defined automatically by reflection.

Reply:
It lets you turn on caching in Dafny's LSP server

Reply:
Oh I see, so it's a Boogie option that needed to be declared by Dafny.